uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/filters/unique.filter.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-env amd */
+
+define([
+  'common/lodash',
+], function (_) {
+  'use strict';
+
+  uniqueFilter.__name = 'unique';
+  uniqueFilter.$inject = ['$filter', '$rootScope', '$log'];
+
+  function uniqueFilter ($filter, $rootScope, $log) {
+    $log.debug('Filter: unique');
+
+    /**
+     * Filters the array to return unique values
+     *
+     * @param  {Array} inputArray
+     * @param  {Array} identifier to filter by
+     * @return {Array}
+     */
+    return function (inputArray, identifier) {
+      if(!inputArray.length) {
```

Comment:
`_.uniq([])` returns `[]`

Why this check is needed?

Reply:
right! not needed!